cscs-checks/tools/profiling_and_debugging/intel_advisor_roofline.py

```diff
@@ -26,52 +26,68 @@ class IntelRooflineTest(rfm.RegressionTest):
     analysis ('advixe-cl -collect survey') and then run a tripcounts+flops
     analysis ('advixe-cl -collect tripcounts -flop') using the same project
     directory for both steps.
+
+    Example result on 1 core of Intel Broadwell CPU (E5-2695 v4):
+        G3_AOS_SCALAR: gflops,  2.79 arithmetic_intensity', 0.166 380ms <- slow
+        G3_AOS_VECTOR: gflops,  3.79 arithmetic_intensity', 0.125 143ms
+        G3_SOA_SCALAR: gflops,  2.79 arithmetic_intensity', 0.166 351ms
+        G3_SOA_VECTOR: gflops, 10.62 arithmetic_intensity', 0.166  57ms <- fast
     '''
     def __init__(self, repeat, toolsversion, datalayout):
         super().__init__()
         self.descr = 'Roofline Analysis test with Intel Advisor'
+        # advisor/2019 is failing on dom ("Exceeded job memory limit")
+        # https://webrt.cscs.ch/Ticket/Display.html?id=36087
```

Comment:
@jgphpc, sorry my lack of knowledge about the issue, but didn't you solve the memory limit on #814?